source/main/assets/tutorial/fr/zettelkasten.md

```diff
@@ -0,0 +1,58 @@
+
+Pour quitter la recherche globale (et faire en sorte que la liste des fichiers ait à nouveau l'air « normale »), cliquez maintenant sur le petit « x » dans le champ de recherche !
+
+## Lier les fichiers 🗄
+
+Il existe deux manières de faire des liens entre les fichiers dans votre Zettelkasten : soit en utilisant un ID, soit en utilisant le nom de fichier (sans extension). Ainsi, si vous avez un fichier appelé « zettelkasten.md », vous pouvez le lier en écrivant `[[zettelkasten]]`. Zettlr essaiera de trouver un fichier avec ce nom de fichier et l'ouvrira.
+
+Mais que se passe-t-il si vous changez le nom du fichier ? Dans ce cas, il est évident que le lien ne fonctionnera plus ! Pour contourner cette limitation, vous pouvez utiliser des ID. Les ID sont simplement des chaînes de chiffres que vous pouvez utiliser pour identifier vos fichiers de manière unique. Vous pouvez ensuite les utiliser pour établir des liens entre vos fichiers. Créons-en un maintenant ! Placez le curseur derrière les deux points et appuyez sur `Cmd/Ctrl+L` :
+
+Désormais, ce fichier a un identifiant que vous pouvez utiliser ! Essayez-le — retournez à l'onglet avec le guide « Bienvenue sur Zettlr ! », et tapez `[[` quelque part. Dans la fenêtre d'autocomplétion, choisissez ce fichier et confirmez votre sélection. Ensuite, `Cmd/Ctrl`, cliquez sur ce même lien pour revenir à ce fichier. Vous remarquerez que Zettlr a lancé une nouvelle recherche, mais surtout : vous pouvez voir les résultats de la recherche en surbrillance ! Ceci est utile pour les liens croisés Zettelkasten, mais sera bien sûr également utile lorsque vous ferez des recherches globales.
+
+
+
+## Utiliser les mots-clés (tags) 🏷
+
+Cela dit, faire des liens n'est pas le seul moyen de créer des relations entre les notes. Vous pouvez également utiliser des mots-clés pour cela. Les mots-clés fonctionnent exactement comme les hashtags sur Twitter, donc vous pouvez #créer #des hashtags #comme #vous #voulez ! En cliquant sur `Cmd/Ctrl`, vous lancerez également une recherche et mettrez en évidence tous les fichiers qui contiennent tel ou tel mot-clé.
+
+Il y a aussi un nuage de mots-clés auquel vous pouvez accéder en cliquant sur l'icône « Mots-clés » dans la barre d'outils. Il énumérera tous vos mots-clés et indiquera le nombre de fichiers qui les utilisent. Vous pouvez filtrer et gérer vos tags à partir de là. Alors que les liens Zettelkasten créent des connexions « dures » entre les fichiers, les tags sont une sorte de connexion « floue » entre des contenus connexes et peuvent vous convenir davantage.
+
+
```

Comment:
Is this personal style or is this common in French texts to leave a little bit more space before headings? Just curious, as I never saw this style before! :)

Reply:
This is a personal style :) I work with different text editor, and different fonts styles and colors. Adding spaces allows me to see the contents more clearly.